qa/httpCalls/submitter/getKeyRotationMessageToSign.py

```diff
@@ -2,12 +2,21 @@
 
 
 # execute a submitter/getKeyRotationMessageToSign call
-def http_get_key_rotation_message_to_sign(sidechain_node, pub_key_to_sign, key_type, withdrawal_epoch):
+def http_get_key_rotation_message_to_sign_for_master_key(sidechain_node, pub_key_to_sign, withdrawal_epoch):
     j = {
         "schnorrPublicKey": pub_key_to_sign,
-        "keyType": key_type,
         "withdrawalEpoch": withdrawal_epoch
     }
     request = json.dumps(j)
-    response = sidechain_node.submitter_getKeyRotationMessageToSign(request)
+    response = sidechain_node.submitter_getKeyRotationMessageToSignForSigningKey(request)
+    return response["result"]
+
+
+def http_get_key_rotation_message_to_sign_for_signing_key(sidechain_node, pub_key_to_sign, withdrawal_epoch):
+    j = {
+        "schnorrPublicKey": pub_key_to_sign,
+        "withdrawalEpoch": withdrawal_epoch
+    }
+    request = json.dumps(j)
+    response = sidechain_node.submitter_getKeyRotationMessageToSignForMasterKey(request)
```

Comment:
the method name and the API endpoint name are different.
http_get_key_rotation_message_to_sign_for_**signing_key**
submitter_getKeyRotationMessageToSignFor**MasterKey**

Same for other method above.